hack/check-renovate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
nit: perhaps having a README-dev.md file with a section about renovate would be more fitting than having this script (or just have both)?